spec/event_store/sink_spec.rb

```diff
@@ -187,6 +187,19 @@ def persisted_tuples_for_aggregate(aggregate_id)
           expect(persisted_tuples_for_aggregate(staged_events.first.aggregate_id).length).to eql 1
         end
       end
+
+      context "when there is a different type of unique constraint violation" do
+        let(:staged_events) {[build_staged_event(aggregate_sequence: 2)]}
+        let(:non_unique_event_sequence_violation) { Sequel::UniqueConstraintViolation.new("this text does not match") }
```

Comment:
Nice naming 👍

Reply:
Thanks for reviewing and approving! Hope you had a nice break!